pcmanfm-qt/pcmanfm-qt_de.ts

```diff
@@ -1537,7 +1540,7 @@ above the folder-view and not above the left pane.</source>
     <message>
         <location filename="../../preferences.ui" line="673"/>
         <source>Needs ffmpegthumbnailer</source>
-        <translation type="unfinished"></translation>
+        <translation>Benötigt ffmpegthumbnailer</translation>
```

Comment:
Probably right. @PCMan, @tsujan Thoughts? Does PCManFM-Qt really still rely on ffmpegthumbnailer?

That aside not really a translation issue. If the string is superfluous it will be removed from the code and hence from the translations automatically as well.